packages/expo-face-detector/src/FaceDetector.ts

```diff
@@ -204,7 +203,15 @@ export async function detectFacesAsync(
   uri: string,
   options: DetectionOptions = {}
 ): Promise<DetectionResult> {
-  if (!ExpoFaceDetector.detectFaces) {
+  if (!ExpoFaceDetector || !ExpoFaceDetector.detectFaces) {
```

Comment:
would be good if we can further check the running environment is expo go. we have the information from [expo-constants](https://docs.expo.dev/versions/latest/sdk/constants/#executionenvironment). but that would require dependency to expo-constants.
does it make sense to you?

Reply:
Even though people already have expo-constants installed (expo depends on it), I would rather leave it as is. Adding a dependency always brings the risk of having multiple versions.

Reply:
checking `global.expo?.modules?.ExponentConstants` is clever 👍

Reply:
checking appOwnership before showing that warn